riotfile.py

```diff
@@ -205,6 +205,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 )
             ],
         ),
+        Venv(
```

Comment:
It doesn't really fit either. I could see this test suite having it's own set of dependencies if we shift to use a different library to encode json. I can also see the number of tests growing significantly. I have no strong feelings either way but it might be cleaner if we run these tests in isolation